### PR TITLE
Add recipe for brie

### PR DESCRIPTION
**Pull request by mcflugen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mcflugen** [comment]
@conda-forge/staged-recipes this recipe is ready for review. Thanks!

File: recipes/baked-brie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - bmipy
+    - numpy
+    - pyyaml
+    - scipy
```
Comment (ocefpaf) [inline recipes/baked-brie/meta.yaml:23]: You probably don't need these at build time, unless the package imports itself while building.

```suggestion
```

File: recipes/baked-brie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UNC-CECL/brie/archive/v{{ version }}.tar.gz
+  sha256: ff48678ed9540276e216d521eeb44fd50db69fd99959c29143455cc38f34afc2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (ocefpaf) [inline recipes/baked-brie/meta.yaml:14]: I'm pretty sure this one can be noarch.

```suggestion
  number: 0
  noarch: python
```

**Comment by mcflugen** [comment]
@ocefpaf Thank you! I _think_ this should be good now.

**Pull request merged**